### PR TITLE
Enh/py37 support

### PR DESCRIPTION
**Pull request by mschmidt87**
This PR adds Python 3.7 to the Continuous Integration and documents support for Python 3.7 in the README file.

Relies on #40 .

### CODE REVIEW

**Comment by jakobj** [comment]
thanks, but conda on travis seems to complain that there's no python 3.7 available. that's strange, do you have any idea why this fails? also please resolve the conflicts generated by merging the other PRs

**Comment by mschmidt87** [comment]
I think the problem was that our CI was based on miniconda2, which apparently doesn't contain python 3.7. I switched it to miniconda3 and now the CI passes.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The user can set two global parameters:
 
 [![Python2.7](https://img.shields.io/badge/python-2.7-blue.svg)](https://www.python.org/downloads/release/python-2714/)
 [![Python3.6](https://img.shields.io/badge/python-3.6-red.svg)](https://www.python.org/downloads/release/python-369/)
+[![Python3.6](https://img.shields.io/badge/python-3.7-red.svg)](https://www.python.org/)
```
Comment (jakobj) [inline README.md]: this should read `Python3.7`

**Review by jakobj — CHANGES_REQUESTED** [review]
just one change, otherwise :+1:

**Review by jakobj — APPROVED** [review]

**Pull request merged**